### ISSUE TITLE
Suggestions to make the data smaller

### ISSUE THREAD

**Issue by arxpoetica** [bcbooks/scriptures-json#1]
Nice work, Ben.

I would suggest removing the URL, since that can easily be created programmatically from already present info in the data.

Also, up to you, but you don't actually need the verse or chapter data items either, since these are arrays, and can also be deduced programmatically. Just a thought. Probably better to leave those in though, for ease of access and grep.

Finally, `reference` is entirely redundant, since `book,` `chapter`, and `verse` are elsewhere, and a reference can be programmatically created quite easily from that information.

**Comment by bencrowder** [comment]
Agreed. I just pushed version 2, removing the URL and adding `lds_slug` fields so the URLs can be constructed without having to create a new mapping table.

I did consider leaving the chapter/verse numbers out, but I figure that being explicit is better here, especially since the array indexes would be off by one. (Simple to fix, but easier to not have to deal with that.)

The references are definitely redundant, yes. I included them for readability, as IDs for verses, and so that simple applications can get the reference without having to do any extra traversal to get the book name. Whether those reasons warrant their inclusion is of course debatable. :) (I went back and forth on it a few times before deciding to leave them in.)
